onnx/cpp2py_export.cc

```diff
@@ -91,21 +91,32 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
       .value("COMMON", OpSchema::SupportType::COMMON)
       .value("EXPERIMENTAL", OpSchema::SupportType::EXPERIMENTAL);
 
-  defs.def("has_schema", [](const std::string& op_type) -> bool {
-    return OpSchemaRegistry::Schema(op_type) != nullptr;
+  defs.def("has_schema", [](const std::string& op_type,
```

Comment:
Need to change the python interface to expose this new function signature https://github.com/onnx/onnx/blob/master/onnx/defs/__init__.py#L13.
Actually I think it's better to just do

```
has_schema = C.has_schema
has = has_schema  # for backward compatibility
```

Reply:
updated